### PR TITLE
Global settings using ConfigPanel

### PR DESCRIPTION
**Pull request by Tagadda**
See https://github.com/YunoHost/yunohost/pull/1459

### CODE REVIEW

**Comment by Tagadda** [comment]
We could add a "Reset all" button at the bottom, even a "Reset" somewhere near each settings

File: app/src/router/routes.js [hunk 1]
```diff
@@ -361,6 +361,15 @@ const routes = [
...
     }
   },
+  {
+    name: 'tool-settings',
+    path: '/tools/settings',
+    component: () => import(/* webpackChunkName: "views/tools/settings" */ '@/views/tool/ToolSettings'),
+    meta: {
+      args: { trad: 'tools_yunohost_settings' },
+      breadcrumb: ['tool-list', 'tool-settings']
+    }
```
Comment (Axolotle) [inline app/src/router/routes.js:371]: Yo ! Sorry another change on config panels has been merged. It adds routing to tabs.
It should only require that you change your route definition to enable it.
Tabs should then be accessible on `/tools/settings/security` or `/tools/settings/other` in your case.

```suggestion
    path: '/tools/settings',
    component: () => import(/* webpackChunkName: "views/tools/settings" */ '@/views/tool/ToolSettings'),
    children: [
      {
        name: 'tool-settings',
        path: ':tabId?',
        component: () => import(/* webpackChunkName: "components/configPanel" */ '@/components/ConfigPanel'),
        props: true,
        meta: {
          routerParams: [],
          args: { trad: 'tools_yunohost_settings' },
          breadcrumb: ['tool-list', 'tool-settings']
        }
      }
    ]
```

**Review by alexAubin — APPROVED** [review]
:+1: :partying_face:

**Review by tituspijean — APPROVED** [review]
❤️

**Pull request merged**